addons/sale_project/views/project_task_views.xml

```diff
@@ -94,12 +94,14 @@
                 <field name="allow_billable" invisible="1"/>
             </xpath>
             <xpath expr="//field[@name='child_ids']/tree/field[@name='partner_id']" position="after">
+                <field name="sale_line_id" optional="hide" options="{'no_create': True}" readonly="1"/>
```

Comment:
Cannot we do the same thing we did in the form view for this field? That is, the user cannot click on this field if he has no access to Sales app?

Something like that:
```suggestion
                <field name="sale_line_id" optional="hide" options="{'no_create': True}" readonly="1" groups="sales_team.group_sale_salesman"/>
                <field name="sale_line_id" optional="hide" options="{'no_open': True}" readonly="1" groups="!sales_team.group_sale_salesman"/>
```